### PR TITLE
Add merlin status feature

### PR DESCRIPTION
**Pull request by lucpeterson**
Given a spec file, prints number of connected workers and tasks on queue server

```
merlin status workflows/feature_demo/feature_demo.yaml

...

[2019-12-09 16:51:14: INFO] collect_queue                  - Workers:          0 - Queued Tasks:          0
[2019-12-09 16:51:14: INFO] default_verify_queue           - Workers:          0 - Queued Tasks:          0
[2019-12-09 16:51:14: INFO] hello_queue                    - Workers:          0 - Queued Tasks:          0
[2019-12-09 16:51:14: INFO] learn_queue                    - Workers:          0 - Queued Tasks:          0
[2019-12-09 16:51:14: INFO] make_samples_queue             - Workers:          0 - Queued Tasks:          0
[2019-12-09 16:51:14: INFO] predict_queue                  - Workers:          0 - Queued Tasks:          0
[2019-12-09 16:51:14: INFO] pyth2_hello                    - Workers:          0 - Queued Tasks:          4
[2019-12-09 16:51:14: INFO] pyth3_q                        - Workers:          0 - Queued Tasks:          1
[2019-12-09 16:51:14: INFO] stop_workers_queue             - Workers:          0 - Queued Tasks:          1
[2019-12-09 16:51:14: INFO] translate_queue                - Workers:          1 - Queued Tasks:          0

```

#25

### CODE REVIEW

**Comment by ben-bay** [comment]
Woah! This is good. I think it might be jsut a `make fix-style` away from passing the CI tests.

**Comment by koning** [comment]
Yes, this looks like very useful info.

File: merlin/main.py [hunk 1]
```diff
@@ -462,6 +473,42 @@ def setup_argparse():
...
+                            Default: %(default)s",
+    )
+    status.add_argument(
+        "--vars",
+        action="store",
+        dest="variables",
+        type=str,
+        nargs="+",
+        default=None,
+        help="Specify desired Merlin variable values to override those found in the specification. Space-delimited. "
```
Comment (ben-bay) [inline merlin/main.py]: Trying to understand why this flag is under the `status` subparser.

File: merlin/study/celeryadapter.py [hunk 2]
```diff
@@ -141,6 +141,28 @@ def query_celery_workers():
...
+
+    connection = app.connection()
+    found_queues = []
+    try:
+        channel = connection.channel()
+        for queue in queues:
+            try:
+                name, jobs, consumers = channel.queue_declare(queue=queue, passive=True)
+                found_queues.append((name, jobs, consumers))
+                LOG.info(f"{name:30} - Workers: {consumers:10} - Queued Tasks: {jobs:10}")
```
Comment (ben-bay) [inline merlin/study/celeryadapter.py]: Still perplexed by these integers.

**Comment by ben-bay** [comment]
TODO: add a `merlin status` cli test -- even if for now it's just expecting `ERROR failed to resolve broker hostname`

File: merlin/study/celeryadapter.py (hunk 2 above)
Comment (koning) [reply merlin/study/celeryadapter.py]: They are the total field sze for the variable to create the nice table effect.

**Pull request merged**